src/main/java/org/fundacionjala/coding/nestor/SortTheOdd.java

```diff
@@ -0,0 +1,30 @@
+package org.fundacionjala.coding.nestor;
+
+
+import java.util.Arrays;
+
+/**
+ * Class in order to sort only odd numbers from integer array.
+ * https://www.codewars.com/kata/sort-the-odd/train/java
+ */
+
+public class SortTheOdd {
+
+  /**
+   * Method to sort ascending odd numbers but even numbers must be on their places.
+   *
+   * @param array to integers.
+   * @return same array with odd elements sorted.
+   */
+  public int[] sortArray(final int[] array) {
+    int[] f = Arrays.stream(array).filter(n -> n % 2 > 0).toArray();
+    Arrays.sort(f);
```

Comment:
instead of using the Arrays.sort method, you can use directly the sorted method in the stream.
```java
int[] sorted = Arrays.stream(array).filter(n -> n % 2 > 0).sorted().toArray();
```